tests/Avalonia.Interactivity.UnitTests/GetRoutedEventsTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System.Collections.Generic;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Xunit;
+
+namespace Avalonia.Interactivity.UnitTests
+{
+    public class GetRoutedEventsTests
+    {
+        [Fact]
+        public void Pointer_Events_Should_Be_Registered()
+        {
+            var expectedEvents = new List<RoutedEvent> { InputElement.PointerPressedEvent, InputElement.PointerReleasedEvent };
+            var registeredEvents = RoutedEvent.GetRoutedEventsOf<InputElement>();
+            Assert.Contains(registeredEvents, expectedEvents.Contains);
+        }
+
+        [Fact]
+        public void ClickEvent_Should_Be_Registered_On_Button()
+        {
+            var expectedEvents = new List<RoutedEvent> { Button.ClickEvent };
+            var registeredEvents = RoutedEvent.GetRoutedEventsOf<Button>();
+            Assert.Contains(registeredEvents, expectedEvents.Contains);
+        }
+
+        [Fact]
+        public void ContentControl_Should_Not_Contain_ClickEvent()
```

Comment:
Nit: This test uses a different naming convention from the others, `ClickEvent_Should_Not_Be_Registered_On_ContentControl` might be better?